provider/source/src/currency/displayname.rs

```diff
@@ -36,10 +40,7 @@ impl DataProvider<CurrencyDisplaynameV1Marker> for crate::SourceDataProvider {
                     currency
                         .display_name
                         .as_deref()
-                        .or(currency.one.as_deref())
-                        .ok_or(DataError::custom(
-                            "No display name or `one` count found for the currency",
-                        ))?
+                        .ok_or(DataError::custom("No display name found for the currency"))?
```

Comment:
this should also be `IdentifierNotFound`

Reply:
done